app/src/main/java/com/waseefakhtar/doseapp/feature/home/viewModel/HomeViewModel.kt

```diff
@@ -0,0 +1,39 @@
+package com.waseefakhtar.doseapp.feature.home.viewModel
```

Comment:
Can we rename all the `viewModel` packages to `viewmodel` to follow the convention?

Reply:
Absolutely, the next push will contain changes to the package names so we adhere to the convention